src/nock/reporter.ts

```diff
@@ -0,0 +1,108 @@
+import { NonPrimitive } from '@superfaceai/one-sdk/dist/internal/interpreter/variables';
```

Comment:
question: can you elaborate why this lives under nock?

Reply:
Because of a bad decision 😄 I'll move it 👍🏻